test/screens/v2/candidate_generator/widgets/cr_departures_test.exs

```diff
@@ -24,8 +24,7 @@ defmodule Screens.V2.CandidateGenerator.Widgets.CRDeparturesTest do
             direction_to_destination: 1,
             wayfinding_arrows: "right-down",
             priority: [1],
-            travel_time_to_destination: "9-12m",
-            show_via_headsigns_message: false
+            travel_time_to_destination: "9-12m"
```

Comment:
Oh geez, this reminds me. I was supposed to update the audio to remove the travel time message. I forgot, that's my bad. Would you mind doing that, if you're not working on other things? [Slack link](https://mbta.slack.com/archives/C039ARUM48H/p1683124188886079)

Reply:
Yeah no problem. Changes have been pushed to this PR.

Reply:
References to `travel_time_to_destination` can be removed like `show_via_headsigns_message`